### PR TITLE
08_my-location-isn-t-on-your-list-what-do-i-do.md

### PR DESCRIPTION
**Pull request by DarklingMuse**
From the work in https://github.com/fetlife/help/pull/84 to update and refine the FAQs into something more concise.

### CODE REVIEW

File: 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # My location isn't on your list, what do I do?
 
-Currently, the list of locations only includes actual cities (don't shoot the messenger). We realize this is annoying and we need to fix it. However, we cannot fix the problem without overhauling the whole thing, so in the meantime, if your location is not listed, please pick the closest city to you until we can get and integrate a more thorough list of locations. Sorry for the inconvenience :(
+Currently, the list of locations only includes actual cities. (Please don't shoot the messenger because that gets messy!) We realize that this is annoying, and that we need to fix it. 
 
-[Where in the World is Carmen Sandiego?](http://en.wikipedia.org/wiki/Carmen_Sandiego "") Boy, do I miss that game.
+Unfortunately, however, we cannot fix the problem without overhauling the whole thing. In the meantime, if your location is not listed, please pick the closest city to you until we can get and integrate a more thorough list of locations.
```
Comment (nielsenramon) [inline 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md:5]: I would change to:

```
However, we cannot fix the problem without overhauling the whole thing. So in the meantime, if your location is not listed, please pick the closest city to you until we can implement an updated list of locations.
```

File: 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+
+We are sorry for the inconvenience. :(
```
Comment (nielsenramon) [inline 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md:7]: `We are` feels so heavy. I would just make it short and cute: `Sorry for the inconvenience! :-(`

File: 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # My location isn't on your list, what do I do?
```
Comment (nielsenramon) [inline 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md:1]: What do you think about: `I can't find my location in the list, what do I do?`

File: 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
...
 
-Currently, the list of locations only includes actual cities (don't shoot the messenger). We realize this is annoying and we need to fix it. However, we cannot fix the problem without overhauling the whole thing, so in the meantime, if your location is not listed, please pick the closest city to you until we can get and integrate a more thorough list of locations. Sorry for the inconvenience :(
+Currently, the list of locations only includes actual cities. (Please don't shoot the messenger because that gets messy!) We realize that this is annoying, and that we need to fix it.
```
Comment (nielsenramon) [inline 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md:3]: What do you think about: `Currently, our list of locations isn't complete and missing quite some cities. We realize this is annoying, and that we need to fix it!`

Comment (sylee-) [reply 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md:3]: I think, "missing some cities"

I'll make that change and see what you think.  "quite some" doesn't go together.  We could change to "a lot of cities" but that doesn't go with our goal of not using words.

File: 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md (hunk 1 above)
Comment (sylee-) [reply 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md]: Updated

File: 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md (hunk 3 above)
Comment (sylee-) [reply 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md]: Updated

File: 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md (hunk 2 above)
Comment (sylee-) [reply 02_profiles/08_my-location-isn-t-on-your-list-what-do-i-do.md]: Updated

**Review by nielsenramon — APPROVED** [review]

**Pull request merged**